example/nginx.ctmpl

```diff
@@ -0,0 +1,64 @@
+# This is an example Nginx configuration template file.
```

Comment:
The radical conformist in me is now wondering if this should go in an `*/etc` dir [as well](https://github.com/tgross/triton-nginx/pull/1/files#r46154561).